community/kernel/src/test/java/org/neo4j/kernel/impl/locking/community/LockManagerImplTest.java

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright (c) 2002-2015 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.kernel.impl.locking.community;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.mockito.Mockito;
+
+import org.neo4j.helpers.collection.Visitor;
+import org.neo4j.kernel.impl.locking.ResourceTypes;
+
+public class LockManagerImplTest
+{
+
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    @Test
+    public void shouldAllowGetReadWriteLocks()
+    {
+        // given
+        LockResource node1 = new LockResource( ResourceTypes.NODE, 1l );
+        LockResource node2 = new LockResource( ResourceTypes.NODE, 2l );
+        LockTransaction lockTransaction = new LockTransaction();
+        LockManagerImpl lockManager = new LockManagerImpl( new RagManager() );
+
+        // expect
+        Assert.assertTrue( lockManager.getReadLock( node1, lockTransaction ) );
+        Assert.assertTrue( lockManager.getReadLock( node2, lockTransaction ) );
+        Assert.assertTrue( lockManager.getWriteLock( node2, lockTransaction ) );
```

Comment:
Not really a fun of static imports, but for junit stuff agree - should be fine to use them. Will update.